package.json

```diff
@@ -31,28 +31,28 @@
     "react-dom": "16.13.1",
     "react-scripts": "3.4.1",
     "roughjs": "4.2.1",
-    "socket.io-client": "2.3.0"
-  },
-  "devDependencies": {
+    "socket.io-client": "2.3.0",
+    "node-sass": "4.14.0",
```

Comment:
I'm curious, why did you move those to dependencies rather than dev dependencies?

Reply:
This was made to get the React app built using `NODE_ENV=production`. It's generally discouraged to install development dependencies inside Docker images as they may cause slow builds and larger images. Sure, the image size is not to worry here since I'm using the multi-stage build, but a bit faster the build process should be, nevertheless.

By default, deploying to Now also installs dev dependencies, so this approach has worked well up to this point.